File: mvp/src/main/java/com/mandria/android/mvp/rx/callbacks/OnNext.java
package com.mandria.android.mvp.rx.callbacks;

import android.support.annotation.NonNull;

import io.reactivex.functions.BiConsumer;

/**
 * Interface for onNext handler.
 */
public interface OnNext<U, V> extends BiConsumer<U, V> {

    @Override
    void accept(@NonNull U u, V v) throws Exception;
}

File: core_kotlin/src/main/kotlin/com/mandria/android/mvp/rx/callbacks/OnNext.kt
package com.mandria.android.mvp.rx.callbacks

import io.reactivex.functions.BiConsumer


/** Interface for onNext handler. **/
interface OnNext<U, V> : BiConsumer<U, V> {

    @Throws(Exception::class)
    override fun accept(u: U, v: V)
}